pkgs/applications/kde/kalgebra.nix

```diff
@@ -0,0 +1,20 @@
+{
+  mkDerivation, lib,
+    extra-cmake-modules, kdoctools,
+    analitza, ki18n, kinit, kirigami2, kconfigwidgets, kwidgetsaddons, kio, kxmlgui, qtwebengine, plasma-framework
+}:
+
+mkDerivation {
+  pname = "kalgebra";
+  outputs = [ "out" "dev" ];
+  meta = {
+    homepage = "https://apps.kde.org/kalgebra/";
+    description = "Graphing calculator";
+    license = with lib.licenses; [ gpl2 ];
+    maintainers = with lib.maintainers; [ ninjafb ];
+  };
```

Comment:
Please keep the `meta` attribute at the end, just like most of the derivations in `nixpkgs`.